### PR TITLE
Fix API keys on the management page

### PR DESCRIPTION
**Pull request by bdmogal**
On-hold because @PGuimarais is going to fix some CSS for the management page in this PR.

### CODE REVIEW

File: cdap-ui/app/cdap/components/Management/index.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ var shortid = require('shortid');
 var classNames = require('classnames');
 
 var dummyData = {
-  version: '4.0-SNAPSHOT',
+  version: '4.0.0',
```
Comment (bdmogal) [inline cdap-ui/app/cdap/components/Management/index.js:36]: this need not be dummy, right? We should use http://docs.cdap.io/cdap/current/en/reference-manual/http-restful-api/configuration.html#cdap-version

Comment (bdmogal) [reply cdap-ui/app/cdap/components/Management/index.js:36]: I guess its unrelated to this PR, and there's more stuff to integrate on this page, so we could do this in a separate PR.

**Comment by ajainarayanan** [comment]
@PGuimarais Seems to be some styling issues with sytem services section. Also I remember you asking about overflow-y but that seems to have been missed.


screen shot 2016-11-23 at 12 16 11 pm
screen shot 2016-11-23 at 12 16 04 pm

**Review by bdmogal — COMMENTED** [review]
lgtm. please squash commits.

**Pull request merged**